livekit-agents/livekit/agents/voice/agent_session.py

```diff
@@ -562,6 +562,9 @@ async def drain(self) -> None:
 
         await self._activity.drain()
 
+    def close_soon(self) -> None:
+        self._close_soon(reason=CloseReason.USER_INITIATED, drain=True)
```

Comment:
for telephony agents, i think this would also need to remove the user participant so their phone hangs up on their end

also I think offering a public close & hangup method is a good idea on its own, so it might make sense to name this method `end_call` instead of `close_soon`, for the developer's benefit

Reply:
it's not just for telephony clients.. SDK clients would also still stay in the room. I think end call is in a 1 on 1 scenario, and the idea is that it ends it for the other side as well. (i.e. delete_room)

Reply:
This will be done with https://linear.app/livekit/issue/AGT-1935/add-delete-room-on-close-to-roomoptions instead